src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/FormattingVisitor.cs

```diff
@@ -7,12 +7,14 @@
 using System.Linq;
 using Microsoft.AspNetCore.Razor.Language;
 using Microsoft.AspNetCore.Razor.Language.Components;
-using Microsoft.AspNetCore.Razor.Language.Extensions;
 using Microsoft.AspNetCore.Razor.Language.Syntax;
-using Microsoft.CodeAnalysis.Razor.Workspaces;
 using Microsoft.CodeAnalysis.Text;
 
-namespace Microsoft.AspNetCore.Razor.LanguageServer.Formatting;
+namespace Microsoft.CodeAnalysis.Razor.Formatting;
+
+using SyntaxNode = Microsoft.AspNetCore.Razor.Language.Syntax.SyntaxNode;
+using SyntaxToken = Microsoft.AspNetCore.Razor.Language.Syntax.SyntaxToken;
+using SyntaxWalker = Microsoft.AspNetCore.Razor.Language.Syntax.SyntaxWalker;
```

Comment:
Consider moving these above the namespace declaration.

Reply:
Done except for SyntaxNode - see my comment above, there is already RazorSyntaxNode